src/Tools/dotnet-user-jwts/src/Commands/DeleteCommand.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.CommandLineUtils;
+using Microsoft.Extensions.Tools.Internal;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+internal sealed class DeleteCommand
+{
+    public static void Register(ProjectCommandLineApplication app)
+    {
+        app.Command("delete", cmd =>
```

Comment:
We use 'remove' in other places in CLI tools, should we be consistent?

Reply:
Yep fair. We can make that change in preview.6